pkg/secrets/asm.go

```diff
@@ -85,23 +84,16 @@ func awsConfigFromParams(params ...interface{}) (*aws.Config, error) {
 	}, nil
 }
 
-func paramsToMap(params ...interface{}) (map[string]string, error) {
+func paramsToMap(params map[string]string) (map[string]string, error) {
```

Comment:
So there's not much happening in this function anymore, just key validation. Could you move the code in `awsConfigFromParams`?

Notice that the type validation in the loop through the map keys should be removed because now we already know that the parameters values are also strings.

Reply:
in particular, this is now useless: https://github.com/ContainerSolutions/externalsecret-operator/pull/26/files#diff-c99f0b857cac6f2378044a5ff415f34cR101

Reply:
Created an issue: https://github.com/ContainerSolutions/externalsecret-operator/issues/28